olive/engine/packaging/sample_code/GenAIOnnxModel/python/code_sample.py

```diff
@@ -108,7 +109,10 @@ def _main():
             if name in args and getattr(args, name)
         }
     )
-    params.set_search_options(search_options)
+    if version.parse(og.__version__) > version.parse("0.1.0"):
+        params.set_search_options(**search_options)
+    else:
```

Comment:
do we really need to support genai 0.1.0? I think it is ok to only support 0.2.0

Reply:
I think that too. But currently 0.1.0 is the official one. 
0.2.0 is in pre-release.

![image](https://github.com/microsoft/Olive/assets/13343117/14997f19-7455-421f-a83c-c9f215b8eb47)

Reply:
we can stop support 0.1.0 since the example need use 0.2.0. otherwise, we need set trust_remote_code or something similar like what Emma run into.

Reply:
Okay, let me update it.

Reply:
agree. let's just support the latest version. their code went through a lot of changes since it was still in pre-release

Reply:
Done.